.gitignore

```diff
@@ -0,0 +1,261 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Correct the gitignore to this one: [VisualStudio.gitignore]( https://github.com/github/gitignore/blob/master/VisualStudio.gitignore)
That will solve some issues I have comments on now.